warehouse/admin/routes.py

```diff
@@ -180,6 +180,13 @@ def includeme(config):
         traverse="/{project_name}/{version}",
         domain=warehouse,
     )
+    config.add_route(
```

Comment:
Would a matching admin button make sense to have generally? I know _most_ of this kind of activity is due to malware reports, but there could be other valid reasons to quarantine something without an existing observation.

Reply:
Certainly can do! I'm thinking at the "Delete project" card at the bottom of the details - unless you think it ought to be somewhere else?

Reply:
Added in 4e76f598a93d62105347a4081bd1cdd14f5dfacf